contracts/contracts/GrantRoundManager.sol

```diff
@@ -65,6 +65,7 @@ contract GrantRoundManager {
    * @notice Creates a new GrantRound
    * @param _owner Grant round owner that has permission to update the metadata pointer
    * @param _payoutAdmin Grant round administrator that has permission to payout the matching pool
+   * @param _matchingToken Address for the matching pool tokens
```

Comment:
If we update the comment for this param above, just making a note to remember to do that here also

Reply:
Addressed [17133de ](https://github.com/dcgtc/dgrants/pull/85/commits/17133de9d645a163a0ebc57efa019b2745e10db3)